src/helpers/contextCreatorHelpers/index.test.js

```diff
@@ -74,6 +76,75 @@ describe('contextCreatorErrorHelper', () => {
     });
   });
 
+  describe('addErrorsAndDataToManifestKeyValuePair', () => {
+    const manifestNoneArrayQuestions = {
+      title: 'Title of page',
+      questions: {
+        question1: {
+          id: 'question1',
+          mainAdvice: 'question 1 main advice',
+        },
+        question2: {
+          id: 'question2',
+          mainAdvice: 'question 1 main advice',
+        },
+        question3: {
+          id: 'question3',
+          mainAdvice: 'question 3 main advice',
+        },
+      },
+      errorMessages: {
+        Field1Required: 'Field 1 is required',
+        Field1TooLong: 'Field 1 is too long',
+        Field2Required: 'Field 2 is required',
+      },
+    };
+    const formattedErrors = formatErrors({ manifest: manifestNoneArrayQuestions, errors });
+    it('should return the contents of the manifest', () => {
+      const manifestWithErrors = addErrorsAndDataToManifestKeyValuePair(
+        { manifest, errors: formattedErrors },
```

Comment:
little confused.. here shouldn't we be passing manifestNoneArrayQuestions to this function instead?

Reply:
yeah we should updated it test still passes think it must be looking at the formatted errors more which was being pased the new array
also updated the name of the manifest